test/com/google/javascript/jscomp/integration/ProductionCoverageInstrumentationPassIntegrationTest.java

```diff
@@ -41,51 +41,19 @@
 public final class ProductionCoverageInstrumentationPassIntegrationTest
```

Comment:
I think we should have a test that references the instrumentation array, even if it's something simple like `console.log(ist_array);`.

Maybe a test that turns on ADVANCED optimizations, too.

Reply:
Added test cases. Also created the test cases in a way where it would show someone how it might work with Advanced optimizations.